src/shared/components/InActivityWarningModal/InActivityWarningModal.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC, useEffect, useState } from 'react';
+import { useIdleTimer } from 'react-idle-timer';
+
+type InActivityWarningModalProps = {
+	onActivity: () => void;
+};
+
+const InActivityWarningModal: FC<InActivityWarningModalProps> = ({ onActivity }) => {
+	const [remainingTime, setRemainingTime] = useState<number>(0);
+	const { getRemainingTime } = useIdleTimer({
+		onAction: onActivity,
+		throttle: 15000,
+		timeout: 60_000,
```

Comment:
stay consistent
```suggestion
		throttle: 15_000,
		timeout: 60_000,
```

Reply:
these get replaced by consts in a later PR